lib/DraggableCore.es6

```diff
@@ -319,18 +320,31 @@ export default class DraggableCore extends React.Component {
     return this.handleDragStop(e);
   };
 
+  removeScroll: EventHandler<MouseEvent> =  (e) => {
+    e.preventDefault();
+    return false;
+  };
+
   // Same as onMouseDown (start drag), but now consider this a touch device.
   onTouchStart: EventHandler<MouseEvent> = (e) => {
     // We're on a touch device now, so change the event handlers
     dragEventFor = eventsFor.touch;
 
+    // Stop the touch device from scrolling while user is dragging.
+    if (this.state.dragging) {
```

Comment:
PS, I initially tried this in handleDragStart and handleDragEnd and had trouble getting it to work. I resorted to the 'dragging' and on touch end events after some trial and error. Perhaps I should try that again, when I get a handle on the addEvent and removeEvent helpers. 

When you say 'the' removeEvent helper, is there a particular one that works with IE ( I assume you mean old version of IE, as I thought the new ones support add/remove event listener)?

We tested on windows touch devises, but I'm not sure which browsers my friend tested. I'll check with her to make sure we are covering FireFox, Chrome, IE, and Safari.